go-controller/pkg/ovn/pods.go

```diff
@@ -26,6 +28,52 @@ import (
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
 )
 
+func (bnc *BaseNetworkController) updateExpectedLogicalPorts(pod *kapi.Pod, lsManager *lsm.LogicalSwitchManager,
```

Comment:
I expect this to have different implementations depending on the topology (i.e. for flat l2 we connect directly to the cluster-wide logical switch instead of the node logical switch.

I think this would be a good opportunity to define how that will look like.